platformtest/harness/harness_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+// Idea taken from
+// https://github.com/openSUSE/umoci/blob/v0.2.1/cmd/umoci/main_test.go
+//
+// How to generate coverage:
+// go test -c -covermode=atomic -cover -coverpkg github.com/zrepl/zrepl/...
+// sudo ../logmockzfs/logzfsenv /tmp/zrepl_platform_test.log \
+//     ./harness.test -test.coverprofile=/tmp/harness.out \
+//     -test.v __DEVEL--i-heard-you-like-tests \
+//     -root rpool/zreplplayground
+//  go tool cover -html=/tmp/harness.out -o /tmp/harness.html
+//
+// Merge with existing coverage reports using gocovmerge:
+//  https://github.com/wadey/gocovmerge
+
+func TestMain(t *testing.T) {
```

Comment:
above stuff needs to be wrapped in a script that can e executed in one line on any OS that supports the Go command